numpy/core/src/_simd/_simd.dispatch.c.src

```diff
@@ -390,6 +390,8 @@ SIMD_IMPL_INTRIN_1(@intrin@_@sfx@, v@sfx@, v@sfx@)
 #if @fp_only@
 SIMD_IMPL_INTRIN_2(max_@sfx@, v@sfx@, v@sfx@, v@sfx@)
 SIMD_IMPL_INTRIN_2(maxp_@sfx@, v@sfx@, v@sfx@, v@sfx@)
+SIMD_IMPL_INTRIN_2(min_@sfx@, v@sfx@, v@sfx@, v@sfx@)
+SIMD_IMPL_INTRIN_2(minp_@sfx@, v@sfx@, v@sfx@, v@sfx@)
```

Comment:
style nit: just above there is a repeat clause for 4 values, here all 4 are spelled out explicitly. Can we unify the approach?

Reply:
maybe a repeater similar to
https://github.com/numpy/numpy/blob/9a889942f7e83d1b7fc474094061c0ab9e41d26b/numpy/core/src/_simd/_simd.dispatch.c.src#L379-L388

Reply:
So let me refactor to this way

Reply:
done